packages/itwinui-react/src/core/Table/types/react-table-types.d.ts

```diff
@@ -0,0 +1,1320 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable @typescript-eslint/no-empty-interface */
+/* eslint-disable @typescript-eslint/ban-types */
+// Type definitions for react-table 7.7
+// Project: https://github.com/tannerlinsley/react-table
+// Definitions by: Guy Gascoigne-Piggford <https://github.com/ggascoigne>,
+//                 Rohit Garg <https://github.com/gargroh>
+//                 Jason Clark <https://github.com/riceboyler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+// reflects react-table@7.7.0
+
+// tslint:disable:no-empty-interface
+// no-empty-interface is disabled to allow easy extension with declaration merging
+
+// TODO: Find out why the below line was giving an error
+// /* eslint-disable @definitelytyped/no-unnecessary-generics */
+// no-unnecessary-generics is disabled because many of these definitions are either used in a generic
+// context or the signatures are required to match for declaration merging
+
+// The changelog for the important changes is located in the Readme.md
```

Comment:
nit: Our changelog is located in readme?

Reply:
This is the file that we copy-pasted from [@types/react-table](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-table/Readme.md). So they are referring to their Readme. I will make this clearer when going through this file again (similar to https://github.com/iTwin/iTwinUI/pull/1514#discussion_r1316004917).